internal/exec/resolvable/resolvable.go

```diff
@@ -362,7 +362,8 @@ func (b *execBuilder) makeFieldExec(typeName string, f *schema.Field, m reflect.
 	var out reflect.Type
 	if methodIndex != -1 {
 		out = m.Type.Out(0)
-		if typeName == "Subscription" && out.Kind() == reflect.Chan {
+		sub, ok := b.schema.EntryPoints["subscription"]
```

Comment:
This is an existing bug which prevents the `subscription` root op having a type named anything other than `Subscription`. Had to fix it here to verify correct behaviour for custom type names, but can split that work if desired